### PR TITLE
Sdk

### PR DESCRIPTION
**Pull request by Ankit098**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @browserstack/afd-dev
```
Comment (karanshah-browserstack) [inline CODEOWNERS:1]: Let's add automate-public-repos which we have for our repos.

File: browserstack.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# BROWSERSTACK_ACCESS_KEY as env variables
+userName: YOUR_USERNAME
+accessKey: YOUR_ACCESS_KEY
+
+# ======================
+# BrowserStack Reporting
+# ======================
+# The following capabilities are used to set up reporting on BrowserStack:
+# Set 'projectName' to the name of your project. Example, Marketing Website
+projectName: BrowserStack Samples
```
Comment (karanshah-browserstack) [inline browserstack.yml:14]: framework: playwright for sending source data to browserstack?

**Comment by karanshah-browserstack** [comment]
README.md updates not done

File: README.md [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
-* Run `npm i`
+* Clone the repo `git clone -b sdk https://github.com/browserstack/node-js-playwright-browserstack.git` and run `cd node-js-playwright-browserstack`.
+* Set `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` as environment variables with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings) or update the same in `browserstack.yml` file.
+* Run `npm i` to install the dependencies.
 
 ## Running your tests
 
-- To run a sample test, run `npm run sample-test`
-
-## Run tests on locally hosted websites
```
Comment (karanshah-browserstack) [inline README.md:17]: Steps for running local steps

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Ankit098) [reply README.md:17]: right now the main command runs all 3 tests.. will sync f2f to disc the approach for demonstrating local vs normal tests..

Comment (Ankit098) [reply README.md:17]: added

**Review by karanshah-browserstack — APPROVED** [review]

**Pull request merged**